issue-tracker/src/routing/Link.tsx

```
@@ -1,37 +1,41 @@
-import RoutingContext from './RoutingContext';
-import React from 'react';
+/* eslint no-restricted-globals:0 */
+import React, { useCallback, useContext } from 'react'
+import RoutingContext from './RoutingContext'
 
-const { useCallback, useContext } = React;
+interface Props {
+  to: string
+  children: React.ReactNode
+}
 
 /**
  * An alternative to react-router's Link component that works with
  * our custom RoutingContext.
  */
-export default function Link(props) {
-  const router = useContext(RoutingContext);
+export default function Link(props: Props) {
+  const router = useContext(RoutingContext)
```

this function requires that the context is set - let's throw an error if it isn't and then remove the null checks (`router?.foo`) below. so something like `if (router == null) { throw new Error('<Link> requires a routing context to be set.') }`